### DISCUSSION TITLE
enable ssl on the ui

### DISCUSSION THREAD

**Discussion by saleemalharir1** [0xJacky/nginx-ui#338] [Q&A]
i just want to ask how to enable https on the UI 
i have everything running but i can only access the UI through http 
do i need to give the certs to nginx from the command line or from inside the UI ?

**Comment by mattiapv** [comment]
Add a new site with this configuration [Example of Nginx Reverse Proxy Configuration](https://github.com/0xJacky/nginx-ui?tab=readme-ov-file#example-of-nginx-reverse-proxy-configuration)

If you are using nginx-ui in a Docker environment this configuration is already present in `/etc/nginx/nginx.conf` , so you will need to remove it from there first.
